### DISCUSSION TITLE
Render texture not able to render after context restoration.

### DISCUSSION THREAD

**Discussion by raj20011990** [pixijs/pixijs#8955] [Q&A]
@ivanpopelyshev. I am using pixi v4 and facing one challenge during pixi context loss restoration. Those texture which are generated through render texture are not able to re-render on screen. Is there any way to work around with this?

**Comment by ivanpopelyshev** [comment]
After restoration you have to re-fill all render textures, because videomemory is the only place where data was stored, there's no extra storage for them

**Comment by raj20011990** [comment]
@ivanpopelyshev, I tried to re-render again those render texture but not working, do you have any example to re-fill those?
